### DISCUSSION TITLE
Feature Requests/Mods

### DISCUSSION THREAD

**Discussion by bwlove319** [theory/sqitch#613] [Ideas]
I just discovered [DBT](https://docs.getdbt.com/docs/building-a-dbt-project/tests).

You may have already addressed this, but what are your thoughts on adding DBT integration?
IIRC Sqitch uses pgTAP.  Has that changed?

**Comment by theory** [comment]
Sqitch does not use pgTAP; you might be thinking of [this](https://justatheory.com/2014/01/templating-tests-with-sqitch/), which is a tutorial for using pgTAP alongside Sqitch, and to have Sqitch scaffold a pgTAP test when you add a change, But Sqitch does not run the tests itself.

For [verify scripts](http://sqitch.org/docs/manual/sqitch-verify/), it simply uses the CLI for the relevant engine. Verify passes if the CLI exits without error, and fails if it exits with an error. It's all SQL.
